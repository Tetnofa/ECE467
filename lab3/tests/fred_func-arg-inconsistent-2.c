bool f1(int a);

// Not an input we need to expect (at most 1 prototype and function name).
// void f1(bool b);

// void f1(int a) {
//   bool b;
//   b = a;
// }

void f1(bool a) {
  int b;
  b = a;
}
