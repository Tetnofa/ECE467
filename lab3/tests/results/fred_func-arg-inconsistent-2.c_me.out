sema: 11:0 : definition inconsistent with earlier definition of f1
sema: 13:2 : type mismatch 
